CMakeLists.txt

```diff
@@ -85,6 +85,26 @@ endif(WITH_CCACHE)
 
 option(WITH_SCCACHE "Build with sccache.")
 if(WITH_SCCACHE)
+  if(NOT FORCE_MAX_JOBS)
+    execute_process(
+      COMMAND "sccache" "--dist-status"
+      OUTPUT_VARIABLE sccache_dist_status
+      RESULT_VARIABLE sccache_dist_status_result
+      COMMAND_ERROR_IS_FATAL ANY
+    )
+    message(STATUS "sccache --dist-status: ${sccache_dist_status}")
+    string(
+      JSON sccache_cores
+      ERROR_VARIABLE sccache_dist_status_parse_error
+      GET "${sccache_dist_status}" SchedulerStatus 1 num_cpus
+    )
+    if (sccache_dist_status_result)
```

Comment:
could you please remove the space after `if` to be more consistent with the other part of this CMake script?